src/test/scala/uk/gov/hmrc/hmrcfrontend/views/helpers/hmrcLayoutSpec.scala

```diff
@@ -438,6 +439,55 @@ class hmrcLayoutSpec extends AnyWordSpecLike with Matchers with JsoupHelpers wit
       backLink.html()       shouldBe "Back Welsh"
     }
 
+    "include the back link if passed a BackLink" in {
+      val hmrcLayout     = app.injector.instanceOf[HmrcLayout]
+      val messages       = getMessages()
+      val customBackLink = BackLink("/some/url", "custom-class", Map("some" -> "attribute"), Text("Custom Back"))
+      val layout         = hmrcLayout(backLink = Some(customBackLink))(
+        Html("")
+      )(fakeRequest, messages)
+      val document       = Jsoup.parse(contentAsString(layout))
+
+      val actualBackLink = document.select(".govuk-back-link")
+      actualBackLink                            should have size 1
+      actualBackLink.attr("href")             shouldBe "/some/url"
+      actualBackLink.hasClass("custom-class") shouldBe true
+      actualBackLink.attr("some")             shouldBe "attribute"
+      actualBackLink.html()                   shouldBe "Custom Back"
+    }
+
+    "add attributes to activate the JavaScript browser back in hmrc-frontend" in {
```

Comment:
this is really a unit test of `BackLink.mimicsBrowserBackButtonViaJavaScript` but feels more coherent/discoverable to have here?